code/datums/diseases/advance/symptoms/beesymptom.dm

```diff
@@ -30,24 +30,24 @@
 		if(2)
 			if(prob(2))
 				to_chat(M, "<span class='notice'>You taste honey in your mouth.</span>")
-		if(3)
-			if(prob(10))
+		if(3, 4, 5)
```

Comment:
As ike told me, we can't do this with switches. We can't call the stages more than once, or it'll just get stuck on this step and won't ever process the other ones.

Reply:
Ah. Well, when you take care of the other issue, please find some way to make this proc at later stages too